### PR TITLE
Test feature

### PR DESCRIPTION
**Pull request by vikitaotiz**
The project tests the us of unit test in javascript using jest.
We have added test for add method
We have added test for remove method

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team 👋,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/util.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import addNewItem from './addTodo.js';
+import removeTodo from './removeTodo.js';
+
+const todoObject = { description: 'New todo', completed: false, index: 1 };
+const todoObject2 = { description: 'New todo 2', completed: false, index: 1 };
+
+require('jest-localstorage-mock');
+/* eslint-disable */
+const todos = localStorage.__STORE__['todos'] || [];
+/* eslint-disable */
```
Comment (mikenath223) [inline src/util.test.js]: - [x] Kindly avoid disabling eslint rules. Rather fix the issue in the code.

File: src/util.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import addNewItem from './addTodo.js';
+import removeTodo from './removeTodo.js';
+
+const todoObject = { description: 'New todo', completed: false, index: 1 };
+const todoObject2 = { description: 'New todo 2', completed: false, index: 1 };
+
+require('jest-localstorage-mock');
```
Comment (mikenath223) [inline src/util.test.js]: - [x] Please use the `es6` `import` syntax instead.

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @vikitaotiz 🙋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 # Required Changes ♻️ 🚧 
Please work on the changes previously requested 🙏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by blakbox23** [comment]
Hello @AshakaE  We've made the required changes in the tests. What exactly are we missing? These are the changes we've made in accordance to the last review.
 These changes include

- [x]  Avoid disabling eslint rules. 
- [x]  Using the es6 import syntax.

**Comment by Davidosky007** [comment]
## Hi @vikitaotiz @blakbox23 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
job-welldone-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**